src/Encoder/Encoder.php

```diff
@@ -47,7 +47,8 @@ final class Encoder
     public static function encode(
         string $content,
         ErrorCorrectionLevel $ecLevel,
-        string $encoding = self::DEFAULT_BYTE_MODE_ECODING
+        string $encoding = self::DEFAULT_BYTE_MODE_ECODING,
+        ?Version $versionNumber = null
```

Comment:
Probably rename this to `$forcedVersion`.

Reply:
I changed the parameter name in both files (Writer and Encoder) to $forcedVersion